backend/experiment/management/commands/templates/experiment.py

```diff
@@ -27,10 +27,14 @@ def __init__(self):
 
     def first_round(self, experiment):
         
-        # 1. Informed consent
-        rendered = render_to_string('consent/consent_new_experiment.html')
-        consent = Consent(rendered, title=_(
-            'Informed consent'), confirm=_('I agree'), deny=_('Stop'))
+        # 1. Informed consent from file or admin (admin has priority)        
+        consent = Consent(
+            experiment.consent, 
+            title=_('Informed consent'), 
+            confirm=_('I agree'), 
+            deny=_('Stop'), 
+            url='consent/consent_new_experiment.html'
```

Comment:
Please change this to `consent/consent.html`, as `consent_new_experiment.html` is not auto-created, and also shouldn't be.